code/common/timer.q

```diff
@@ -1,23 +1,22 @@
 // Functionality to extend the timer
 
 \d .timer
-
 enabled:@[value;`enabled;1b]				// whether the timer is enabled
 debug:@[value;`debug;0b]				// print when the timer runs any function
-logcall:@[value;`logcall;1b]				// log each timer call by passing it through the 0 handle
+logcall:(not @[value;`.proc.lowpowermode;0b]) & @[value;`logcall;1b]				// log each timer call by passing it through the 0 handle
```

Comment:
Comment is way out in space due to tabs.

Reply:
yes line up the comments please